addons/website_forum/populate/forum_post.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+from odoo import fields, models
+from odoo.tools import populate
+
+QA_WEIGHTS = {0: 25, 1: 35, 2: 20, 3: 10, 4: 4, 5: 3, 6: 2, 7: 1}
+_logger = logging.getLogger(__name__)
+
+class Post(models.Model):
+    _inherit = 'forum.post'
+    # Include an additional average of 2 post answers for each given size
+    # e.g.: 100 posts as populated_model_records = ~300 actual forum.posts records
+    _populate_sizes = {'small': 100, 'medium': 1000, 'large': 90000}
+    _populate_dependencies = ['forum.forum', 'res.users']
+
+    def _populate_factories(self):
+        forum_ids = self.env.registry.populated_models['forum.forum']
```

Comment:
We can here define a seeed for random, so we always populate the same way. 
See https://github.com/odoo/odoo/pull/139150/files#diff-02e881d3b2b12393dd0ad71209a4942c99b68011f64903c5ecb1c33046fa8f16R16